pkgs/modules/run-package-json/default.nix

```
@@ -0,0 +1,19 @@
+{ runPackageJsonScript, runFileScript }:
+
+{ config, lib, pkgs-unstable, ... }:
+
+let
+  bun = pkgs-unstable.callPackage ../../bun { };
+
+  script = pkgs-unstable.writeScript "package-json-runner" ''
+    ${bun}/bin/bun ${./script.js}
+  '';
+in
+
+{
+  replit.runners."package.json" = {
+    name = "package.json";
+    start = "${script}/bin/package-json-runner --run-script ${runFileScript} --run-package-json-script ${runPackageJsonScript}";
```

What about this instead (might need some slight tweaking)?
```suggestion
    start = "${bun}/bin/bun ${./script.js} --run-script ${runFileScript} --run-package-json-script ${runPackageJsonScript}";
```

---

🤦 that should be just `${script}`. but invoking the script like in your suggestion will make users more scared to change their run configs - i'm trying to avoid that with the 2nd script. lemme try the shebang again

---

should be better now?

---

Oh yeah, I hadn't thought about that showing up in the Console. Looks fine.